libraries/microsoft-agents-activity/microsoft_agents/activity/channels.py

```diff
@@ -10,6 +10,18 @@ class Channels(str, Enum):
     Ids of channels supported by ABS.
     """
 
+    """Agents channel."""
+    agents = "agents"
+    agents_email_sub_channel = "email"
```

Comment:
Not a PR blocker: We ended up pulling the Agents subchannel definitions out of DotNet SDK.  Kept the parent "agents" though.  This is because these are also defined in the A365 Extension.